doc/source/parameters/cfd/post_processing.rst

```diff
@@ -18,8 +18,12 @@ This subsection controls the post-processing other than the forces and torque on
     set kinetic energy name              = kinetic_energy
 
     # Average velocities calculation
-    set calculate average velocities     = false
-    set initial time                     = 0.0
+    set calculate average velocities      = false
+    set initial time for average velocity = 0.0
+
+    # Average temperature calculation
+    set calculate average temperature and average heat flux        = false
+    set initial time for average temperature and average heat flux = 0.0
```

Comment:
```suggestion
    set calculate average temperature and average heat flux        = false
    set initial time for average temperature and average heat flux = 0.0
```

average temperature and heat flux would be sufficient no? Although I understand you do not want to be ambiguous.... What do you think?